### DISCUSSION TITLE
Why does petite-vue show this as a raw text

### DISCUSSION THREAD

**Discussion by AhzDeveloperlol** [vuejs/petite-vue#195] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/70972043/207370132-abc86aa9-e6ae-41a6-ac9c-3bd55dd27969.png)

this is the code:

```
<div v-scope="{ count: 0 }">
        {{ count }}
        <button @click="count++">inc</button>
      </div>
```

**Comment by sqllyw** [comment, marked as answer]
did you have this line before the code:

    <script src="https://unpkg.com/petite-vue" defer init></script>

https://jsfiddle.net/yz2mhon6/

**Reply by AhzDeveloperlol** [reply]
sorry it was a mistake in the url so it couldnt load
